app/src/main/java/me/quadphase/qpdex/databaseAccess/PokemonFactory.java

```diff
@@ -1122,6 +1196,79 @@ public void loadAllTypes() {
     }
 
     /**
+     * Loads all the Abilities into memory. Should be done at program start (during splash screen)
+     */
+    public void loadAllAbilities() {
+        // fail safe 0th index of types:
+        abilities[0] = new Ability("None","None");
```

Comment:
Consider naming this "Ability" since it is unclear what each of the filters are.